test/e2e/setup/setup.go

```diff
@@ -164,10 +169,33 @@ func deployOperator() error {
 	); err != nil {
 		return errors.Errorf("error building operator deployment: %s", err)
 	}
+
+	if err := wait.PollImmediate(time.Second, time.Duration(30)*time.Second, hasDeploymentRequiredReplicas(dep)); err != nil {
```

Comment:
in this repo, I see in most of the cases we're using this format: `30 * time.Second` instead of `time.Duration(30)*time.Second`

Reply:
fixed! :)